content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -8,66 +8,80 @@ card:
+The name of a `ConfigMap` object must be a valid
 [DNS subdomain name](/docs/concepts/overview/working-with-objects/names#dns-subdomain-names).
 
-When you are creating a ConfigMap based on a file, the key in the \<data-source> defaults to the basename of the file, and the value defaults to the file content.
+When you are creating a `ConfigMap` based on a file, the key in the \<data-source>
+defaults to the basename of the file, and the value defaults to the file content.
 
-You can use [`kubectl describe`](/docs/reference/generated/kubectl/kubectl-commands/#describe) or
-[`kubectl get`](/docs/reference/generated/kubectl/kubectl-commands/#get) to retrieve information
-about a ConfigMap.
+You can use
+[`kubectl describe`](/docs/reference/generated/kubectl/kubectl-commands/#describe) or
+[`kubectl get`](/docs/reference/generated/kubectl/kubectl-commands/#get) to retrieve
+information about a `ConfigMap`.
 
-#### Create ConfigMaps from directories
+### Create a ConfigMap from a directory
 
-You can use `kubectl create configmap` to create a ConfigMap from multiple files in the same directory. When you are creating a ConfigMap based on a directory, kubectl identifies files whose basename is a valid key in the directory and packages each of those files into the new ConfigMap. Any directory entries except regular files are ignored (e.g. subdirectories, symlinks, devices, pipes, etc).
+You can use `kubectl create configmap` to create a `ConfigMap` from multiple files in
+the same directory. When you are creating a `ConfigMap` based on a directory, the
+`kubectl` tool identifies files whose basename is a valid key in the directory,
+then packages each of those input files into the new `ConfigMap`.
+Each input must be a regular file; the `kubectl` tool ignores paths that refer to any
+other kind of item (such as symbolic links, directories, devices, or pipes).
 
 For example:
 
 ```shell
-# Create the local directory
+# Create a local directory
 mkdir -p configure-pod-container/configmap/
 
-# Download the sample files into `configure-pod-container/configmap/` directory
+# Download some sample files into the directory "configure-pod-container/configmap/"
 wget https://kubernetes.io/examples/configmap/game.properties -O configure-pod-container/configmap/game.properties
 wget https://kubernetes.io/examples/configmap/ui.properties -O configure-pod-container/configmap/ui.properties
 
-# Create the configmap
+# Create the ConfigMap
 kubectl create configmap game-config --from-file=configure-pod-container/configmap/
 ```
 
-The above command packages each file, in this case, `game.properties` and `ui.properties` in the `configure-pod-container/configmap/` directory into the game-config ConfigMap. You can display details of the ConfigMap using the following command:
+The above command packages each file, in this case, `game.properties` and `ui.properties`
+in the `configure-pod-container/configmap/` directory into the _game-config_ `ConfigMap`.
```

Comment:
nit: Lines 76-77 could move to line 61 or just before the command instructions? This avoids telling the reader
to refer to "the above command"